prime-router/src/main/kotlin/common/HttpClientUtils.kt

```diff
@@ -521,7 +549,7 @@ class HttpClientUtils {
                     jsonPayload?.let {
                         setBody(jsonPayload)
                     }
-                }
+                }.also { httpClient?.close() }
```

Comment:
I think this is the biggest value add of this ticket.

Reply:
MEMOERY LEAKS HATE THIS ONE SIMPLE TRICK